app/src/main/java/org/mifos/selfserviceapp/presenters/LoanApplicationPresenter.java

```diff
@@ -63,13 +64,17 @@ public void onError(Throwable e) {
                     @Override
                     public void onNext(LoanTemplate loanTemplate) {
                         getMvpView().hideProgress();
-                        getMvpView().showLoanTemplate(loanTemplate);
+                        if (loanState == LoanState.CREATE) {
```

Comment:
@dilpreet96 I think, Template would be same in both cases? So why two separate methods